index.bs

```diff
@@ -499,104 +479,70 @@ When this method is invoked, the user agent MUST execute the following algorithm
     : {{origin}}
     :: The [=unicode serialization of an origin|unicode serialization=] of |rpId|
     : {{hashAlg}}
-    :: UA-chosen
-
-        Issue(w3c/webauthn#362): We need *some* constraints on the possible hash
-        algorithms, or else sites will fail on unusual UAs.
+    :: The [=recognized algorithm name=] of the hash algorithm selected by the client for generating the
+        [=hash of the serialized client data=]
     : {{tokenBinding}}
     :: The [=Token Binding ID=] associated with |callerOrigin| (if any)
-
-        Issue(w3c/webauthn#360): Make sure this association was set up properly.
     : {{ClientData/extensions}}
     :: |clientExtensions|
 
-1. Let |clientDataJSON| be the [=UTF-8 encoding=] of the result of calling the
-    initial value of {{JSON/stringify|JSON.stringify}} on |clientData|.
+1. Let |clientDataJSON| be the [=JSON-serialized client data=] constructed from |clientData|.
 
-    Issue: Some extensions contain ArrayBuffers, which don't stringify well.
-    What's the intent here?
-1. Let |clientDataHash| be the hash of |clientDataJSON| using
-    <code>|clientData|.{{hashAlg}}</code>.
+1. Let |clientDataHash| be the [=hash of the serialized client data=] represented by |clientDataJSON|.
```

Comment:
Yes, that is what it is. The intention of this change was to centralize this definition so all the places in the spec that refer to it can be consistent. I'm fine with updating this to be less ambiguous, I just want to make sure we do the same with all such instances in the spec.